README.md

```diff
@@ -18,4 +18,16 @@ Then you need to run mkdocs from the root of that repository:
 
     $ mkdocs build
 
+Sometimes this might fail on Fedora 24+, so perform below steps
```

Comment:
I'll have to check this. At first glance, these steps seem unnecessary. Can you retry with just a "." for `docs_dir` field. ?